src/runtime_src/core/common/query_requests.h

```diff
@@ -32,6 +32,15 @@ namespace xrt_core {
 
 namespace query {
 
+enum bdf_index
```

Comment:
please define an enum class and make all enumerations lower case:
```
enum class bdf {
domain,
bus,
dev,
...
}
```
and calling it would look something like:
`std::get<bdf::bus>(value)`